docs/src/main/asciidoc/health-indicator.adoc

```diff
@@ -0,0 +1,9 @@
+## Health Indicator
+
+The client library comes with a Health Indicator that check whether the connection to the Azure App Configuration store(s) is healthy or not. If enabled, for each store it gives a status of:
+
+ * UP - The last connection was successful
+ * DOWN- The last connection resulted in a non 200 error code
+ * NOT LOADED - The config store is listed in the local configuration file, but wasn't loaded from at startup. This can either be because it is disabled, or failed to load at startup and the `fail-fast` configuration for the store was set to false.
```

Comment:
do you mean "from the file"? 
this is not clear. are the reasons this can happen because 
1. it is disabled 
2. failed to load at startup and the `fail-fast` configuration for the store was set to false
or because 
1. it is disabled 
2. failed to load at startup
 and in both situations the `fail-fast` configuration for the store was set to false

Reply:
1 & 2, any ideas how to clear this up?

Reply:
Maybe this: 
The config store is listed in the local configuration file, but the config wasn't loaded from the file at startup. This can be because of two reasons. Either loading is disabled, which can be fixed by ___x__y__z_. Otherwise, config failed to load at startup while the `fail-fast` configuration for the store was set to false. This can be fixed by ______setting failfast to true using x command____.

Reply:
@AlexandraKemperMS NOT LOADED isn't a failure condition.

When we say loading is disabled it means someone went into the config file and set enabled to false, there is nothing to fix.

fail-fast is mainly meant for testing or having a primary and backup store at startup. If the primary store fails the secondary store is used, but because of this the primary store is never used even on refresh.

The point being there is nothing to "fix". This is more informational.

Reply:
Ah, I see, then you can remove the parts around fixing it I mentioned above, but keep the two reasons.